### DISCUSSION TITLE
h264 video acceleration in //gpu, but not well

### DISCUSSION THREAD

**Discussion by DocS788** [ChristopherKlay/StadiaEnhanced#171] [General]
Hi,

i see in the "browser://gpu" that video h264 is supported by the device:

<html>
<body>

Encode h264 baseline | 0x0 to 1920x1088 pixels, and/or 30.000 fps
-- | --
Encode h264 main | 0x0 to 1920x1088 pixels, and/or 30.000 fps
Encode h264 high | 0x0 to 1920x1088 pixels, and/or 30.000 fps

</body>
</html>

But when i select it in the codec of stadia, the connection suffers a bit. The codec vp9 goes ok, anyhow my (old) gpu can not suppors neither vp9 or h264...i think that all the work is made by cpu/software.

My question is: i have an old PC, why does h264 is less supported then vp9 ?

**Comment by ChristopherKlay** [comment]
H264 is pretty commonly not well enough supported (i.e. 30fps limitations) while VP9 is optimized enough to work fine on older CPU's by now, but hardware support for GPU's came later on ad most older GPU's don't support decoding it.
